src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const styles = sourceString
+    .split(';')
+    .filter((style) => style.trim().length > 0);
+  const styleOBJ = {};
```

Comment:
The variable name 'styleOBJ' is not very descriptive. Consider using a name that better reflects the content of the variable, such as 'styleObject' or 'cssStyles'.